app/renderer/js/utils/link-util.js

```diff
@@ -35,11 +35,12 @@ class LinkUtil {
 		return isImageUrl.test(url);
 	}
 
-	isPDF(url) {
-		// test for pdf extension
-		const isPDFUrl = /\.(pdf)\?*.*$/i;
-		return isPDFUrl.test(url);
-	}
+	// Code to show pdf in a new BrowserWindow (currently commented out due to bug-upstream)
```

Comment:
Commenting this part of the code doesn't seem to be needed. Others are fine for me.

Reply:
ok sure.. i'll uncomment it